index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Happy International Women’s Day</title>
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+    <section class="page-header">
+        <div class="first-header">
+            <h1>Happy International Women’s Day</h1>
+        </div>
+    
+        <section class="input">
+            <input id="addName" type="text" name="addName" placeholder="Enter name of any woman you know">
+            <button id="addNameToList"><span>Add</span></button>
+        </section>
+    
+        <section class="output">
+            <p class="output-header"> List of Awesome Women I know</p>
+                <div class id="outputList"> 
+                    <ol class id="ordered-list">
+                        <p><li>My mother</li></p>
```

Comment:
Why are you hard-coding this?

Reply:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/35523688/225373109-1a889865-a644-4fa2-bdbc-e1e27647ab08.png">